src/plugins/vis_type_vega/public/vega_fn.ts

```diff
@@ -56,7 +68,10 @@ export const createVegaFn = (
     },
   },
   async fn(input, args, context) {
-    const vegaRequestHandler = createVegaRequestHandler(dependencies, context.abortSignal);
+    // Temporary solution. we should find a more legal way to set custom adapters;
+    context.inspectorAdapters = createInspectorAdapters();
```

Comment:
@ppisljar  currently we don't have any legal way to register custom set of inspector panels for our classical visualizations 

Could you please help me to find better way for that